introduction/development/working-with-symbol-files/how-to-manually-upload-symbols.md

```diff
@@ -1,26 +1,30 @@
 # How to Manually Upload Symbols
 
-## Step 1
+[Symbols](../../../education/bugsplat-terminology.md#symbols) are used to display the original function names and line numbers in source code from the [call stacks](../../../education/bugsplat-terminology.md#call-stack) of compiled applications.
```

Comment:
Suggest:  ...used to display the original source code function names and line numbers from the call stack...